lib/mini_sql_multisite_connection.rb

```diff
@@ -36,6 +36,7 @@ def committed!(*)
 
     def before_committed!(*); end
     def rolledback!(*); end
+    def trigger_transactional_callbacks?; end
```

Comment:
is "no transactional callbacks" the default we are after here?

Reply:
Good question, I think that to stay safe, we should default that to true. I checked how it is used in ActiveRecord and it is used to rollback transaction:
```
def trigger_transactional_callbacks? # :nodoc:
  (@_new_record_before_last_commit || _trigger_update_callback) && persisted? ||
    _trigger_destroy_callback && destroyed?
end

def rollback_records
  return unless records
  ite = records.uniq(&:object_id)
  already_run_callbacks = {}
  while record = ite.shift
    trigger_callbacks = record.trigger_transactional_callbacks?
    should_run_callbacks = !already_run_callbacks[record] && trigger_callbacks
    already_run_callbacks[record] ||= trigger_callbacks
    record.rolledback!(force_restore_state: full_rollback?, should_run_callbacks: should_run_callbacks)
  end
  ...
end
```